### PR TITLE
Update gh-pages branch for rapids 22.02.0 release[skip ci]

### PR DESCRIPTION
**Pull request by nvliyuan**
Update gh-pages for rapids 22.02.0 release[skip ci]
https://nvliyuan.github.io/yuali-spark-rapids

### CODE REVIEW

**Comment by nvliyuan** [comment]
sorry about #4717 merged by mistake, please review the gh-pages update pr again.

**Review by jlowe — COMMENTED** [review]
Please update the PR summary to better explain what this PR is doing.  "Gh pages 2202" is a pretty cryptic title for someone browsing PRs.

File: docs/additional-functionality/rapids-udfs.md [hunk 1]
```diff
@@ -141,19 +141,19 @@ in the [udf-examples](https://github.com/NVIDIA/spark-rapids/tree/main/udf-examp
 
 - [URLDecode](https://github.com/NVIDIA/spark-rapids/tree/main/udf-examples/src/main/scala/com/nvidia/spark/rapids/udf/scala/URLDecode.scala)
 decodes URL-encoded strings using the
-[Java APIs of RAPIDS cudf](https://docs.rapids.ai/api/cudf-java/stable)
+[Java APIs of RAPIDS cudf](https://docs.rapids.ai/api/cudf-java/legacy)
```
[the remark this answers is not in the document]
Comment (nvliyuan) [reply docs/additional-functionality/rapids-udfs.md]: Thanks, add the missing changes

**Review by jlowe — APPROVED** [review]

**Comment by nvliyuan** [comment]
squash and merge this pr since plugin v22.02.0 is released now

**Pull request merged**